generate_certs.go

```diff
@@ -0,0 +1,78 @@
+package main
```

Comment:
Can you explain the reason for adding this go code? 

I believe the intention is to generate a certificate at the server startup. We can generate certificates during dev mode start (`make run`) and server packaging, However shouldn't generate them at each server startup. Otherwise it will not allow configuring custom certificates.

Currently we're generating certificates from the `build.sh` via the make commands at dev mode startup and also packaging to the distribution.
Check: https://github.com/asgardeo/thunder/blob/05706f63b70e6160be62a028b4f7e8c01fd523da/build.sh#L341

You can do a similar implementation in the ps1 file.